WcaOnRails/app/views/registrations/_register_form.html.erb

```diff
@@ -24,6 +24,7 @@
         <% if @registration.new_or_deleted? %>
           <%= f.submit t('registrations.register'), class: "btn btn-success" %>
         <% else %>
+          <%= f.hidden_field :status, value: @registration.checked_status %>
```

Comment:
@viroulep Depending on the implementation in the backend I think it might now be possible for a user to set its own registration to `"accepted"`.

Reply:
We used it to build the registration params [here](https://github.com/thewca/worldcubeassociation.org/blob/b3de7e1960a00fefc7861c05b495786adccf7d69/WcaOnRails/app/controllers/registrations_controller.rb#L314-L327), but it's wrapped in a `can_manage_competition?` so I think we're fine!